ui/src/main/webapp/sources.js

```diff
@@ -0,0 +1,153 @@
+import { gql } from 'react-apollo'
```

Comment:
I don't know where this file should live, but I don't think it should be at the root.